### ISSUE TITLE
Broken links: Release bundle not available

### ISSUE THREAD

**Issue by akcom** [BIDData/BIDMach#141]
Both the wiki (https://github.com/BIDData/BIDMach/wiki/Installing-and-Running#Installing_a_Release_Bundle) and the readme (https://github.com/BIDData/BIDMach/blob/master/README.md) link to http://bid2.berkeley.edu/bid-data-project/download/ which is a broken link.  It appears that BID project now lives at http://bid.berkeley.edu/BIDdata/ but this is mostly just a placeholder website with no direct links to the executable bundle.

**Comment by DanielTakeshi** [comment]
Good point. Another thing is that the current `README.md` refers to a `./getdevlibs.sh` but as [John pointed out in this GitHub issue][1], that command is outdated. I'll update the README. At this point it's probably easier to simply download the latest version and compile with maven.

(I'll leave this issue open in case John wants to comment more on it later.)

[1]:https://github.com/BIDData/BIDMach/issues/139

**Comment by DanielTakeshi** [comment]
Oh, and by the way, to *compile*, you can modify the code, and then run `mvn package` which will compile things. That serves the same function as the previous `./sbt package` command we used to use.
